### PR TITLE
Git clone commands in installation article

### PR DESCRIPTION
**Pull request by Naylin15**
### Changes description
Add folder name to git commands to rename the folder directly, from the Telegram chat Flyve MDM Dev

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [x] Docs have been added/updated

### References

Closes #N/A

### CODE REVIEW

File: _posts/static/2018-02-01-installation-wizard.md [hunk 1]
```diff
@@ -80,10 +80,10 @@ Flyve MDM v2.0.0 runs on GLPI 9.2.1 or later. It depends on inventory features o
   * Using Git, for those who are familiar with it, be aware of the version:
 
     ```console
-      git clone https://github.com/fusioninventory/fusioninventory-for-glpi.git
+      git clone https://github.com/fusioninventory/fusioninventory-for-glpi.git fusioninventory
```
Comment (btry) [inline _posts/static/2018-02-01-installation-wizard.md]: specify the command must run in the plugins folder of GLPI

File: _posts/static/2018-02-01-installation-wizard.md [hunk 2]
```diff
@@ -98,10 +98,10 @@ For more information, check the [Fusion Inventory Documentation](http://fusionin
   * Using Git:
 
     ```console
-      git clone https://github.com/flyve-mdm/glpi-plugin.git
+      git clone https://github.com/flyve-mdm/glpi-plugin.git flyvemdm
```
Comment (btry) [inline _posts/static/2018-02-01-installation-wizard.md]: specify the command must run in the plugins folder of GLPI

File: _posts/static/2018-02-01-installation-wizard.md [hunk 3]
```diff
@@ -80,10 +80,10 @@ Flyve MDM v2.0.0 runs on GLPI 9.2.1 or later. It depends on inventory features o
   * Using Git, for those who are familiar with it, be aware of the version:
 
     ```console
-      git clone https://github.com/fusioninventory/fusioninventory-for-glpi.git
+      git clone https://github.com/fusioninventory/fusioninventory-for-glpi.git fusioninventory
     ```
 * Go to glpi/plugins/ and unpack the file there
-* Rename the folder to ```fusioninventory```
+* Rename the folder to ```fusioninventory``` if needed.
```
Comment (btry) [inline _posts/static/2018-02-01-installation-wizard.md]: renaming the folder becomes useless if the plugin is deployed with GIT.

File: _posts/static/2018-02-01-installation-wizard.md [hunk 4]
```diff
@@ -98,10 +98,10 @@ For more information, check the [Fusion Inventory Documentation](http://fusionin
   * Using Git:
 
     ```console
-      git clone https://github.com/flyve-mdm/glpi-plugin.git
+      git clone https://github.com/flyve-mdm/glpi-plugin.git flyvemdm
     ```
 * Go to glpi/plugins and unzip the file there
-* Rename the folder to ```flyvemdm```
+* Rename the folder to ```flyvemdm``` for zip file download method.
```
Comment (btry) [inline _posts/static/2018-02-01-installation-wizard.md]: renaming the folder becomes useless if the plugin is deployed with GIT.

File: _posts/static/2018-02-01-installation-wizard.md (hunk 3 above)
Comment (Naylin15) [reply _posts/static/2018-02-01-installation-wizard.md]: Done, I removed the rename steps and moved the glpi/plugins location before starting the downloads

**Pull request merged**